100.en.adoc

```diff
@@ -2,116 +2,118 @@
+Several studies show that, for water samples, analyses based on eDNA may have a higher probability of finding rare and cryptic species than conventional methods (https://doi.org/10.1111/j.1365-294X.2011.05418.x[Thomsen et al. 2012^]; https://doi.org/10.1016/j.biocon.2014.11.029[Biggs et al. 2015^]; https://doi.org/10.1111/mec.13428[Valentini et al. 2016^]; https://doi.org/10.1002/edn3.74[Bessey et al. 2020^]). The same may be true in other environmentals. Therefore, eDNA may be suitable for monitoring rare red list species and undesirable alien species that often have low densities and that are difficult to detect with conventional methods. Environmental DNA methods are able to detect cryptic organisms, especially those that are small and unable to be detected by the naked eye (e.g. bacteria and fungi). In addition, eDNA can also be used for observation of many species simultaneously, and may describe entire biological communities or major components of them (https://ntnuopen.ntnu.no/ntnu-xmlui/handle/11250/2612638[Ekrem & Majaneva 2019^]).
 
-Identification and classification of organisms from sequence data and marker-based surveys is dependent upon a reference library of known organisms to match the newly generated sequences to. The efficacy of classification depends on the completeness (coverage) and the reliability of reference libraries, as well as the tools used to carry out the classification. All of these are moving targets, making taxonomic expertise and caution important qualities in the assessment of the results (<<taxonomy-of-sequences,§2.6>>). Availability of ASVs is fundamental for subsequent re-identifications and improvements of identification accuracy.
+Identification and classification of organisms from sequence data and marker-based surveys depends on access to a reference library of sequences taken from known organisms against which to match the newly generated sequences. The efficacy of classification depends on the completeness (coverage) and the reliability of reference libraries, as well as the tools used to carry out the classification. These are all moving targets, making it essential to apply taxonomic expertise and caution in the interpreting results (<<taxonomy-of-sequences,§2.6>>). Inclusion of all verified amplicon sequence variants (ASVs; https://doi.org/10.1038/ismej.2017.119[Сallahan et al. 2017^]) in the reference library will enhance identification accuracy and allow for future reinterpretation of data.
```

Comment:
- '…taken from known organisms that are matched against the newly generated sequences.'
- adjust placement of hyperlink text for ASV ref